src/ophyd_async/epics/adcore/_core_io.py

```diff
@@ -70,14 +70,14 @@ class NDPluginStatsIO(NDPluginBaseIO):
     # Basic statistics
     compute_statistics: A[SignalRW[bool], PvSuffix.rbv("ComputeStatistics")]
     bgd_width: A[SignalRW[int], PvSuffix.rbv("BgdWidth")]
-    total_array: A[SignalRW[float], PvSuffix.rbv("TotalArray")]
+    total_array: A[SignalRW[float], PvSuffix("TotalArray", "Total_RBV")]
```

Comment:
```suggestion
    total_array: A[SignalR[float], PvSuffix.rbv("Total")]
```

Total_RBV is the AI record, TotalArray is the mca, going by the others we want to support the ai